.gitignore

```diff
@@ -19,3 +19,4 @@ yarn-error.log
 # SVG sprite
 /templates/includes/base_icons.html
 **/__pycache__
+.DS_Store
```

Comment:
Minor, but might be good to use of [these two](https://github.com/texastribune/texastribune/blob/need-for-speed/.gitignore#L2-L3) strategies for those pesky DS_Store files.